inference/benchmark_video.py

```diff
@@ -156,7 +158,7 @@ def get_args():
         "--model_id",
         type=str,
         default="THUDM/CogVideoX-5b",
-        choices=["THUDM/CogVideoX-2b", "THUDM/CogVideoX-5b"],
+        # choices=["THUDM/CogVideoX-2b", "THUDM/CogVideoX-5b"],
```

Comment:
Why?

Reply:
I'll undo this later. I'm using a local checkpoint path